README.md

```diff
@@ -1,24 +1,26 @@
 [![Build Status](https://travis-ci.org/IBM/watson-conversation-slots-intro.svg?branch=master)](https://travis-ci.org/IBM/watson-conversation-slots-intro)
-![Bluemix Deployments](https://metrics-tracker.mybluemix.net/stats/8cbdd782337103af2cd09faf13a2481b/badge.svg)
+![IBM Cloud Deployments](https://metrics-tracker.mybluemix.net/stats/8cbdd782337103af2cd09faf13a2481b/badge.svg)
 
 # Creating a Pizza ordering Chatbot using Watson Conversation Slots feature
 
-In this developer journey, we will use the Watson Conversation Slots feature to
+In this developer Code Pattern, we will use the Watson Conversation Slots feature to
```

Comment:
I would drop the "developer" here for consistency, but it is not bad to keep it in here otherwise.